lte/gateway/c/oai/tasks/amf/amf_config.c

```diff
@@ -141,7 +152,646 @@ void amf_config_init(amf_config_t* config) {
 ***************************************************************************/
 int amf_config_parse_opt_line(
     int argc, char* argv[],
-    amf_config_t* config_pP) {  // This function will be removed in upcoming PR
-  amf_config_init(config_pP);
+    amf_config_t* config_aA) {  // This function will be removed in upcoming PR
+  amf_config_init(config_aA);
+  return 0;
+}
+
+/***************************************************************************
+**                                                                        **
+** Name:    amf_config_parse_opt_line()                                   **
+**                                                                        **
+** Description: Invokes amf_config_init() to initialize                   **
+**              default values of AMF                                     **
+**                                                                        **
+**                                                                        **
+***************************************************************************/
+int amf_config_parse_file(amf_config_t* config_aA) {
+  config_t cfg                  = {0};
+  config_setting_t* setting_mme = NULL;
+  config_setting_t* setting     = NULL;
+  config_setting_t* subsetting  = NULL;
+  config_setting_t* sub2setting = NULL;
+  int aint                      = 0;
+  int i = 0, n = 0, stop_index = 0, num = 0;
+  const char* astring = NULL;
+  const char* tac     = NULL;
+  const char* mcc     = NULL;
+  const char* mnc     = NULL;
+#if !EMBEDDED_SGW
+  char* sgw_ip_address_for_s11 = NULL;
+#endif
+  bool swap                = false;
+  const char* imsi_prefix  = NULL;
+  const char* apn_override = NULL;
+
+  config_init(&cfg);
+
+  if (config_aA->config_file != NULL) {
+    /*
+     * Read the file. If there is an error, report it and exit.
+     */
+    if (!config_read_file(&cfg, bdata(config_aA->config_file))) {
+      OAILOG_CRITICAL(
+          LOG_CONFIG, "Failed to parse MME configuration file: %s:%d - %s\n",
```

Comment:
log statement indicates MME configuration file. is AMF configuration put up in MME config file?

Reply:
Comments addressed.